src/internal/tester/config.go

```diff
@@ -111,35 +111,35 @@ func readTestConfig() (map[string]string, error) {
 		TestCfgUserID,
 		os.Getenv(EnvCorsoM365TestUserID),
 		vpr.GetString(TestCfgUserID),
-		"pradeepg@8qzvrj.onmicrosoft.com",
+		"LynneR@10rqc2.onmicrosoft.com",
 	)
 	fallbackTo(
 		testEnv,
 		TestCfgSecondaryUserID,
 		os.Getenv(EnvCorsoSecondaryM365TestUserID),
 		vpr.GetString(TestCfgSecondaryUserID),
-		"lidiah@8qzvrj.onmicrosoft.com",
+		"AdeleV@10rqc2.onmicrosoft.com",
 	)
 	fallbackTo(
 		testEnv,
 		TestCfgLoadTestUserID,
 		os.Getenv(EnvCorsoM365LoadTestUserID),
 		vpr.GetString(TestCfgLoadTestUserID),
-		"leeg@8qzvrj.onmicrosoft.com",
+		"leeg@10rqc2.onmicrosoft.com",
 	)
 	fallbackTo(
 		testEnv,
 		TestCfgLoadTestOrgUsers,
 		os.Getenv(EnvCorsoM365LoadTestOrgUsers),
 		vpr.GetString(TestCfgLoadTestOrgUsers),
-		"lidiah@8qzvrj.onmicrosoft.com,pradeepg@8qzvrj.onmicrosoft.com",
+		"AdeleV@10rqc2.onmicrosoft.com,LynneR@10rqc2.onmicrosoft.com",
```

Comment:
Vivek is claiming AdeleV for the moment.  Can we swap this to JoniS?

Reply:
Going to stick with this one - optimizing for getting this in and not resetting testing